src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Startlist/StartlistHistory.razor

```diff
@@ -0,0 +1,45 @@
+﻿@using NTS.Domain.Core.Objects
+@using NTS.Domain.Objects
+@using NTS.Domain.Setup.Entities
+@using NTS.Judge.Blazor.Ports
+@using System.Collections
+@using Not.Blazor.Components
+@using Not.Blazor.TM.Models
+@using Not.Extensions
+
+@inherits StartlistTabs
+
+@if (_starts.Count() != 0)
+{
+    <NotTrueCenterText Typo="Typo.h5" Content="History" Class="mb-4" />
+    <MudTabs PanelClass="px-4 py-6" Elevation="0" Rounded ApplyEffectsToContainer>
+        @foreach (var tabHeader in _startlistByStage.Keys)
+        {
+            <MudTabPanel Text="@tabHeader">
+                <ChildContent>
+                    <NSimpleTable T="Start" Headings="_tableHeadings" Items="_startlistByStage[tabHeader]"/>
+                </ChildContent>
+            </MudTabPanel>
+        }
+    </MudTabs>
+}
+
+@code{
+    [Inject]
+    public IStartlistHistory _behind { get; set; } = default!;
+
+    protected override IEnumerable<Start> _starts => _behind.History;
```

Comment:
You don't actually need this abstract property, you cans imply pass the _behind.History as argument to `CreateHeadersAndGroupByStage`.